polarCoordinateTraversal.m

```diff
@@ -146,19 +146,19 @@
 radial_voxels = [current_voxel_ID_r];
 
 % III. TRAVERSAL PHASE
+tol = 10^-10;
```

Comment:
Still not a huge fan of adding some magic number here. If we're not using some default tolerance like ```eps```, it should be up to the client to determine it, no? Especially since if they decide to change it here, they may forget to change it in radial_hit. 

In either case, a comment should be added noting both that there is a tolerance ```tol``` and why its necessary.